### DISCUSSION TITLE
I can't sort anymore my 4 pinned discussions since few days ☹️Bug ? or recent change ?

### DISCUSSION THREAD

**Discussion by Guilllllaume** [community/community#54439] [Discussions]
### Select Topic Area

Bug

### Body

Hi!

I used to pin 4 discussions of my main repo and maintain an order of the pinned discussions but since few days i couldn't sort them ☹️

For example: on the left, i could display 2 announcements and 2 Q&A ont the right side 
When a new annoucement was created, i unpinned an old one and i pinned the new and i could put the new one the left but not anymore!

Now the 4 pinned discussions seem to be ordered by creation date.
I can't change that, i can't move and swap the location of pinned discussions.

Is this behavior a new bug ?
or did i miss any updates ?

Please help !

**Comment by Guilllllaume** [comment]
bump

**Comment by markcmiller86** [comment]
I think this is the same request...I used to have a small button that I could use to drag pinned discussions to set my own desired left-to-right ordering for them. I can't anymore. Why did we loose this feature? We can apparently still do the same thing for pinned repos on an org/user page.

**Comment by LindseyB** [comment]
This is a recent change, because our data was showing not a lot of people were using it and there were accessibility problems with the dragging code. 

Apologies for those folks that were using it, but there's no plans at the moment to add it back.

**Comment by pljones** [comment]
Yes, another plus 1 on this.  We have a couple of long term annoucements and one or two short term announcements and adjusting the order is essential to keep the most important prominent.

**Comment by dakshithadissanayaka** [comment]
The inability to sort pinned discussions is indeed a bug, and it's currently being investigated by GitHub. The expected behavior is that users should be able to freely order their pinned discussions, regardless of their creation date.

Here's a summary of the issue:

- Pinned discussions are not sortable.
- Pinned discussions appear to be ordered by creation date, which is not the desired behavior.
- Users are unable to change the order of pinned discussions.

This bug is affecting users who want to maintain a specific order of pinned discussions, especially those with a mix of announcements and Q&A threads. It's important to be able to control the visibility and positioning of pinned discussions to effectively manage the flow of information and discussions within a repository.

GitHub has acknowledged the bug and is working on a fix. In the meantime, users can work around the issue by creating separate issues and pinning them to achieve the desired order. However, this workaround is not ideal as it requires additional steps and doesn't provide the same level of control as the original pinning feature.

Stay tuned for updates from GitHub regarding the fix for this bug. Hopefully, the ability to sort pinned discussions will be restored soon, allowing users to manage their pinned discussions effectively.
